src/libse/SubtitleFormats/PacUnicode.cs

```diff
@@ -124,8 +124,8 @@ private static Paragraph GetPacParagraph(ref int index, byte[] buffer)
             {
                 return null; // probably not correct index
             }
-
-            int maxIndex = timeStartIndex + 10 + textLength;
+            
+            int maxIndex = timeStartIndex + 11 + textLength;
```

Comment:
Just made a quick test - this seems to give a lot of lines ending with `.` - even some already ending with `.` or `!`...

Reply:
Thanks will investigate further as it is 100% chopping off content in certain paragraphs.

Reply:
So running a further test on an FPC (Item2) and PAC(Item1) side by side in English and Burmese we are 1 character short if I reset Line 128 back to 10, however as you can see there are extra dots on the FPC Paragraph 1 & 3 (Item 2), these are 0xFF in the files, which i'm continuing to dig into, its also adding extra space on the second line of Paragraph 1.

The test files were generated in WinCaps which is the Broadstream / Screen Subtitling current commercial Subtitle editor.

I'll continue digging.

![image](https://user-images.githubusercontent.com/32466972/115143120-5554c480-a03d-11eb-83d9-3d4fd1328f5d.png)

Reply:
Kept it extend to 11 as the full stops are present in the file at the correct byte, next to a control character which signifies a line break or paragraph change.

I'm not sure how we can check or change this.

If you count forward from TimeStartIindex 10 you end up with 03 (ETX) control character, instead of the first character of the text, 11 corrects that, then adds the text length to it. 

Please correct me if I'm wrong, I may have gotten completely the complete wrong end of the stick.

Thanks,
Tom